fairmq/plugins/Control.cxx

```diff
@@ -36,6 +37,8 @@ namespace
 
     extern "C" auto sigterm_handler(int signal) -> void
     {
+        ++gSignalCount;
```

Comment:
I agree on this line. This was missing in #266. But I don't see why the rest is necessary.

Reply:
because I was not sure if you wanted to distinguish between term and int. it's fine if we simply increment gSignalCount. Do you want me to change it?

Reply:
Yes, unless I am missing something, incrementing the count is sufficient. If you see any other issue, please elaborate. Otherwise update with the increment only. Thanks!

Reply:
Ok. Done.